db/migrate/20240815183449_enable_legacy_form_embed_for_all_forms.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+# Feature flag migration to accompany release of USWDS-based modal
+class EnableLegacyFormEmbedForAllForms < ActiveRecord::Migration[7.1]
+  def up
+    execute 'UPDATE forms SET legacy_form_embed = TRUE'
+  end
+
+  def down
```

Comment:
This is a legitimate revert because I just verified that `legacy_form_embed` is false for all forms in production.

Reply:
👍